doc/quick-start.md

```diff
@@ -41,53 +43,59 @@ Load the extension:
 %load_ext sql
 ```
 
-Let's see an example using a SQLite database. Let's insert some sample data from the TIOBE index:
+Let's download some sample `.csv` data:
 
 ```{code-cell} ipython3
-%%sql sqlite://
-CREATE TABLE languages (name, rating, change);
-INSERT INTO languages VALUES ('Python', 14.44, 2.48);
-INSERT INTO languages VALUES ('C', 13.13, 1.50);
-INSERT INTO languages VALUES ('Java', 11.59, 0.40);
-INSERT INTO languages VALUES ('C++', 10.00, 1.98);
+from pathlib import Path
+from urllib.request import urlretrieve
+
+if not Path("penguins.csv").is_file():
+    urlretrieve("https://raw.githubusercontent.com/mwaskom/seaborn-data/master/penguins.csv",
+                "penguins.csv")
 ```
 
-## Querying
+Start a DuckDB in-memory database:
 
 ```{code-cell} ipython3
-%sql SELECT * FROM languages
+%sql duckdb://
```

Comment:
We can also show naming connections and having 2 connections at once which is super annoying on native IDEs.

Reply:
I think adding two connections is too much or the quick start; but I added a tip admonition explaining the `--alias` function and linking to the proper section.